### ISSUE TITLE
[Fix] Doesn't run on Mesa: Illegal use of reserved word `switch'

### ISSUE THREAD

**Issue by rversteegen** [azagaya/laigter#37]
Laigter doesn't work for me on Mesa because it fails to ask for GLSL 1.30; Mesa is defaulting to something lower than that.

On starting it I get the error
```
QOpenGLShader::compile(Fragment): 0:106(2): error: illegal use of reserved word `switch'
0:106(2): error: syntax error, unexpected ERROR_TOK
<dump of the shader source code>

QOpenGLShaderProgram::attributeLocation(aPos): shader program is not linked
QOpenGLShaderProgram::attributeLocation(aPos): shader program is not linked
QOpenGLShaderProgram::attributeLocation(aTexCoord): shader program is not linked
QOpenGLShaderProgram::attributeLocation(aTexCoord): shader program is not linked
```
And opening any image just shows a white rectangle.

`switch` was added in GLSL 1.30 (OpenGL 3.0). `glxinfo` shows "OpenGL core profile version string: 3.3 (Core Profile) Mesa 19.3.2".

I [found a thread](http://red.planetarena.org/forum/showthread.php?tid=1851&pid=13697#pid13697) where someone wrote about this error message:
> OK, I get it. Switch/case didn't get added until GLSL 1.30, whereas we're specifying GLSL 1.20. The Nvidia GLSL compilers were allowing it anyway but the Mesa compilers correctly identified this as invalid.

So I added `#version 130` to fshader.glsl and that fixed it. Not submitting a patch because I don't know how you prefer to request GLSL version. I don't know what I'm doing.

**Comment by azagaya** [comment]
Hi! Thanks for the report!
I forgot that switch was not present in 1.20. That switch was introduced las week.
I prefer to keep 1.20 because some users run Laigter on really old systems, which don't support 130 or above. 
If you want, you can make a PR changing the switch statement by a bunch of `if... else if... else...`! If not, please tell me and ill change myself.
Thanks a lot!

**Comment by rversteegen** [comment]
Ah. When I saw the error initially was worried my GPU was too old, it's great to hear you're supporting old systems. (Very nice tool BTW!)

I'll leave it to you to make that change.

**Comment by azagaya** [comment]
Fixed in  7b13cdb

**Comment by azagaya** [comment]
Thanks for the report!
